packages/@okta/vuepress-site/docs/release-notes/2022/index.md

```diff
@@ -2,6 +2,50 @@
 title: Okta API Products release notes 2022
 ---
 
+## August
+
+### Monthly release 2022.08.0
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [PKCE validation for OIDC app integrations is GA in Preview](#pkce-validation-for-oidc-app-integrations-is-ga-in-preview) | July 7, 2022|
```

Comment:
The order of the entries seems random. Didn't we consider ordering them with GA first? Or by date they hit the preview orgs?